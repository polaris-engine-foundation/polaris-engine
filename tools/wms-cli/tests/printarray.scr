func main() {
    for(i in 0..9) {
        hoge[i] = "array" + i;
    }

    for(i in 0..9) {
        print("" + i + "番目の配列の中身は" + hoge[i]);
    }
}
