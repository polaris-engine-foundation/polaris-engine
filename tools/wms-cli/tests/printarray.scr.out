0番目の配列の中身はarray0
1番目の配列の中身はarray1
2番目の配列の中身はarray2
3番目の配列の中身はarray3
4番目の配列の中身はarray4
5番目の配列の中身はarray5
6番目の配列の中身はarray6
7番目の配列の中身はarray7
8番目の配列の中身はarray8
9番目の配列の中身はarray9
